### ISSUE TITLE
Optimizing gold usage for XP

### ISSUE THREAD

**Issue by Vesox33** [TeamFightTacticsBots/Alune#86]
The bot should have an option to buy XP only after reaching a certain gold threshold, e.g., 30 gold (config file?), and only then purchase XP. This is because, otherwise, it often lacks money for units.

**Comment by akshualy** [comment]
Duplicate of #52 
tl;dr this requires an OCR implementation first
but you can also lower the frequency at which the bot buys XP in the config.
